core/src/tui_core/styles/stylesheet.rs

```diff
@@ -73,3 +73,63 @@ impl Stylesheet {
     }
   }
 }
+
+/// Macro to make building `StyleSheet` easy
```

Comment:
Change to something like `Macro to make building [Stylesheet] easy.`

Why?

It is good to leverage Rust's ability to generate great documentation. Taking a symbol that is in scope (eg `Stylesheet`) and surrounding it w/ `[ ]` makes it linkable in the docs generated for docs.rs. 

You can also see the docs that are generated on your computer as you're making changes using `cargo doc --open`.